bot/exts/events/advent_of_code/_cog.py

```diff
@@ -235,6 +216,43 @@ async def aoc_leaderboard(
         await view.wait()
         await message.edit(view=None)
 
+    @in_month(Month.DECEMBER)
+    @adventofcode_group.command(
+        name="leaderboard",
+        aliases=("board", "lb"),
+        brief="Get a snapshot of the PyDis private AoC leaderboard",
+    )
+    @whitelist_override(channels=AOC_WHITELIST_RESTRICTED)
+    async def aoc_leaderboard(
+            self,
+            ctx: commands.Context,
+            self_placement_name: Optional[str] = None,
+    ) -> None:
+        """
+        Get the current top scorers of the Python Discord Leaderboard.
+
+        Additionaly you can specify a `self_placement_name`
```

Comment:
Small typo:

```suggestion
        Additionally you can specify a `self_placement_name`
```

Reply:
Thank you for the quick review! 
Sorry, I should've made the `_format_leaderboard`  more consistent.
Will push a patch ASAP.

Reply:
Also, I noticed a minor problem, in the code.
I have to modify the cache a little bit. 
Should be fine!